worlds/ror2/__init__.py

```diff
@@ -116,9 +116,9 @@ def fill_slot_data(self):
     def create_item(self, name: str) -> Item:
         item_id = item_table[name]
         item = RiskOfRainItem(name, ItemClassification.filler, item_id, self.player)
-        if name == 'Dio\'s Best Friend':
+        if name == "Dio\'s Best Friend":
```

Comment:
pretty sure that should be
```suggestion
        if name == "Dio's Best Friend":
```
?

Reply:
Uh I thought it would let me accept your suggestion when I clicked that sorry. Blame github mobile